src/TheCallCenter/Data/Entities/Call.cs

```diff
@@ -9,16 +10,23 @@ namespace TheCallCenter.Data.Entities
 {
     public class Call
     {
-    public int Id { get; set; }
-    [Required]
-    public string Name { get; set; }
-    [Required]
-    [EmailAddress]
-    public string Email { get; set; }
-    [Required]
-    public string Problem { get; set; }
-    public DateTime CallTime { get; set; } = DateTime.UtcNow;
-    public bool Answered { get; set; } = false;
-    public DateTime AnswerTime { get; set; } = DateTime.MinValue;
-  }
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+
+        [Required]
+        public string Name { get; set; }
+
+        [Required]
+        [EmailAddress]
+        public string Email { get; set; }
+
+        [Required]
+        public string Problem { get; set; }
```

Comment:
If you're going to move this to .NET 6, we should move it to Nullable Reference Types too. (See the csproj changes.)